crates/astria-bridge-withdrawer/src/bridge_withdrawer/startup.rs

```diff
@@ -0,0 +1,613 @@
+use std::{
+    sync::Arc,
+    time::Duration,
+};
+
+use astria_core::{
+    bridge::Ics20WithdrawalFromRollupMemo,
+    primitive::v1::asset,
+    protocol::{
+        asset::v1alpha1::AllowedFeeAssetIdsResponse,
+        bridge::v1alpha1::BridgeAccountLastTxHashResponse,
+        transaction::v1alpha1::Action,
+    },
+};
+use astria_eyre::eyre::{
+    self,
+    ensure,
+    eyre,
+    Context as _,
+    ContextCompat,
+    OptionExt as _,
+};
+use prost::Message as _;
+use sequencer_client::{
```

Comment:
Avoid mixing the sequencer_client reexports and the types imported from `astria-core.` Reexporting all these types from `astria-sequencer-client` was probably an error.

Reply:
this causes a cascade of compilation errors where functions expect e.g. `sequencer_client::Address` instead of `core::generate::Address`

Reply:
You should almost never use the types defined in `generate`. Use the ones from, for example, `astria_core::primitiv`, not `astria_core::generated::primitive`